web/thesauruses/_meta/lists.json

```diff
@@ -24,12 +22,8 @@
       "name_of_sorted_mutable_list": "What is a sorted mutable list called?",
       "create_a_sorted_mutable_list": "Create the list",
       "sorted_mutable_list_start_number": "What number does it start at?",
-      "sorted_mutable_list_can_be_appended": "Can you append to it?",
-      "sorted_mutable_list_can_be_inserted_in_middle": "Can you insert into the middle of it?",
       "access_element_in_sorted_mutable_list": "Access element by index",
-      "insert_into_beginning_of_sorted_mutable_list": "Insert element at beginning",
-      "insert_into_end_of_sorted_mutable_list": "Insert element at end",
-      "insert_into_middle_of_sorted_mutable_list": "Insert element in middle",
+      "insert_into_sorted_mutable_list": "Insert element in middle",
```

Comment:
For sorted/unordered lists insertion at a specific position is meaningless, hence we can join them into one.

Reply:
Also makes sense.